bin/cjson-parser.c

```diff
@@ -0,0 +1,106 @@
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <cjson/cJSON.h>
+
+void readJSONFile(const char *filename);
+void iterJSONFile(cJSON *json);
+
+int main() {
+    const char *filename = "config.json";
+    readJSONFile(filename);
+    return 0;
+}
+
+void iterJSONFile(cJSON *json) {
+ 
+    cJSON *proc_key = cJSON_GetObjectItemCaseSensitive(json, "process");
+    cJSON *root_key = cJSON_GetObjectItemCaseSensitive(json, "root");
+
+    json = json->child;
+
+    while (json != NULL) {
+        if (strcmp(json->string, "ociVersion") == 0) {
+            // validate oci Version
+            printf("ociVersion: %s\n", json->valuestring);
+        } else if (strcmp(json->string, "process") == 0) {
+            // child: env - set environment to PATH value, cwd corresponds to --cd
+            cJSON *env_key = cJSON_GetObjectItemCaseSensitive(proc_key, "env");
+            printf("process env: %s\n", env_key->child->valuestring);
```

Comment:
This seems risky because `env_key->child` could be a null pointer if the `config.json` is malformed.